src/set.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+export default class Set {
+  constructor(){
+    this._elements = []
+  }
+
+  size() {
+    return this._elements.length
+  }
+
+  add( element ) {
+    if (!this.contains( element )) { this._elements.push ( element ) }
+  }
+
+  contains( element ) {
+    return this._elements.includes( element )
+  }
+
+  isEmpty() {
+    return this.size() === 0
+  }
+
+  remove( element ) {
+    if ( this.contains( element )) {
+      const index = this._elements.indexOf( element )
+      this._elements.splice( index, 1 )
+    }
+  }
+
+  forEach( callbackFunction ) {
+    this._elements.map( callbackFunction )
+  }
```

Comment:
Defining your own forEach function for this class helps to keep the code more readable in the other functions. Great work.